images/spark/Dockerfile

```diff
@@ -34,6 +34,10 @@ WORKDIR ${SPARK_HOME}
 RUN rm jars/kubernetes-client-4.1.2.jar
 ADD https://repo1.maven.org/maven2/io/fabric8/kubernetes-client/4.4.2/kubernetes-client-4.4.2.jar jars
 
+# Jars for S3 access support in case of Spark History Server
+ADD http://central.maven.org/maven2/org/apache/hadoop/hadoop-aws/2.7.3/hadoop-aws-2.7.3.jar jars
+ADD http://central.maven.org/maven2/com/amazonaws/aws-java-sdk/1.7.4/aws-java-sdk-1.7.4.jar jars
```

Comment:
IMO it's better to include them in our Spark distribution tarball, but it'snot a showstopper, we can address it later.